core-ui/src/components/Extensibility/ExtensibilityDetails.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import { usePrepareDetailsProps } from 'resources/helpers';
+import { ResourceDetails } from 'shared/components/ResourceDetails/ResourceDetails';
+import { prettifyNamePlural } from 'shared/utils/helpers';
+
+import { useGetCRbyPath } from './useGetCRbyPath';
+import { Widget } from './components/Widget';
+import { useGetTranslation, TranslationBundleContext } from './helpers';
+
+export const ExtensibilityDetailsCore = ({ resMetaData }) => {
+  const { widgetT } = useGetTranslation();
+
+  const detailsProps = usePrepareDetailsProps(
+    resMetaData.navigation.path,
+    resMetaData.navigation.label,
+  );
+
+  if (resMetaData.resource.kind) {
+    detailsProps.resourceUrl = detailsProps.resourceUrl.replace(
+      resMetaData.navigation.path,
+      resMetaData.resource.kind.toLowerCase(),
+    );
+  }
+
+  const header = resMetaData?.details?.header || [];
+  const body = resMetaData?.details?.body || [];
+  const schema = resMetaData?.schema;
+
+  const breadcrumbs = [
+    {
+      name: resMetaData.navigation.label || resMetaData.navigation.path,
+      path: '/',
+      fromContext: resMetaData.navigation.path,
+    },
+    { name: '' },
+  ];
+  return (
+    <TranslationBundleContext.Provider value={resMetaData.navigation.path}>
```

Comment:
I guess we can remove it from this component, as it's already used in `ExtensibilityDetails` below.

Reply:
yes, absolutely. good catch. even more, that's what the extra component was created for :)